gate-oauth2/src/main/groovy/com/netflix/spinnaker/gate/security/oauth2/SpinnakerUserInfoTokenServices.groovy

```diff
@@ -99,6 +117,10 @@ class SpinnakerUserInfoTokenServices implements ResourceServerTokenServices {
     def username = details[userInfoMapping.username] as String
     def roles = []
 
+    if(!userRoleInfoUri.isEmpty()) {
+      roles = getUserInfo(accessToken)[userRoleInfoMapping] as ArrayList
```

Comment:
Can this ever result in `roles = null` ? If so, it could be troublesome downstream.